packages/client/clientSchema.graphql

```diff
@@ -104,3 +104,8 @@ extend type Discussion {
   isAnonymousComment: Boolean!
   replyingToCommentId: String!
 }
+
+extend type PollOption {
+  placeholder: String
+  shouldAutoFocus: Boolean
```

Comment:
This looks pretty bad as it's mixing the implementation detail with a data layer. The reason I've added is, when the user clicks `Add another option`, the new option has to be automatically focused. On the other hand, when the user initially creates a poll, the poll title has to be focused so we need a way to figure out if the option is the one initially added, or added as a result of a user action. If you have some ideas on how to improve that, I'd love to do that!